benchmarks/crossvalidation.py

```diff
@@ -0,0 +1,26 @@
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import GridSearchCV, cross_val_score
+
+from .common import Benchmark
+from .datasets import _synth_classification_dataset
+
+
+class CrossValidation_bench(Benchmark):
+    def setup(self, params):
+        self.n_jobs = params[0]
+        self.X, self.y = _synth_classification_dataset(n_samples=70000,
+                                                       n_features=200,
+                                                       random_state=0)
+
+    def time_crossval(self, params):
+        clf = RandomForestClassifier(n_estimators=100, random_state=0)
```

Comment:
Please also add a peakmem function. This is important for the cross validation tools.